appengine/flexible_python37_and_earlier/tasks/README.md

```diff
@@ -106,10 +106,15 @@ location is "us-central1").
 ```
 export LOCATION_ID=us-central1
 ```
+
 ### Using App Engine Queues
+
 Running the sample will create a task, targeted at the `/example_task_handler`
 endpoint, with a payload specified:
 
+NB: please update `create_app_engine_queue_task.py` before running the following
```

Comment:
Please consider changing to `Note:` or `Important:`. `NB` meaning may not be clear for everyone - as non native speaker I had to Google it ;-)

Reply:
On that note, you can also use the Github markdown syntax: https://github.com/orgs/community/discussions/16925

Reply:
That's a good call @rsamborski XD, thank you!
@kweinmeister good to know! Updated.